### PR TITLE
Angel/video process failure

### PR DESCRIPTION
**Pull request by angellee-io**

### CODE REVIEW

File: src/com/grideo/videoProcessing/models/JobStatusNotification.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+public class JobStatusNotification {
+
+    @JsonIgnoreProperties(ignoreUnknown=true)
+    public static class JobInput {
```
Comment (aaljebouri) [inline src/com/grideo/videoProcessing/models/JobStatusNotification.java]: Why static class and why not into it's own separate file? Just wondering.

File: src/com/grideo/videoProcessing/requestHandlers/VideoProcessingRequestHandler.java [hunk 2]
```diff
@@ -30,83 +29,88 @@
...
+	private void updateVideoProcessToFailAndRemoveOrphanFiles(JobStatusNotification notification) {
+		updateVideoProcessStatus(VideoProcessStatus.FAILED, transcoderHelper.getFileIdFromJobStatusNotification(notification));
+		removeOrphanFiles(notification);
 	}
-
-	private Boolean updateVideoToProcessCompleted(String fileId) {
-		LOG.info("updateToProcessCompleted");
-		LOG.info("DatabaseManager: " + databaseManager);
+	
+	private Boolean updateVideoProcessStatus(VideoProcessStatus status, String fileId) {
```
Comment (aaljebouri) [inline src/com/grideo/videoProcessing/requestHandlers/VideoProcessingRequestHandler.java]: If this is only ever is used for success do you need to pass the status in?

Comment (angellee-io) [reply src/com/grideo/videoProcessing/requestHandlers/VideoProcessingRequestHandler.java]: It is also being used to update the status to failed

File: src/com/grideo/videoProcessing/models/JobStatusNotification.java (hunk 1 above)
Comment (angellee-io) [reply src/com/grideo/videoProcessing/models/JobStatusNotification.java]: ah I grab this class from aws. I don't know why tbh.

**Pull request merged**